src/dstack/_internal/server/migrations/versions/27d3e55759fa_add_pools.py

```diff
@@ -83,6 +83,7 @@ def upgrade() -> None:
                 "NEBIUS",
                 "TENSORDOCK",
                 "VASTAI",
+                "VULTR",
```

Comment:
You do not edit existing migrations since there will be no effect if the migrations are applied  – you need to add a new migration.

In this specific case, the migrations have to be generated for postgres. You need to set up a local postgres database, configure dstack to use it (e.g. `export DSTACK_DATABASE_URL="postgresql+asyncpg://postgres:postgres@localhost:5432/postgres"`), and then generate migrations with alembic (`alembic revision --autogenerate` from the server dir).

Reply:
Remove migrations edits and try adding a new migration. If you think you need help, let me know – I'll add the postgres migrations before merging.

Reply:
Got it. I will do the needful and push.

Reply:
@r4victor I've added a new migration. Please share your feedback or suggestions. If further updates are needed, I’ll make the necessary changes and push again.